www/include/views/graphs/graphs.html

```diff
@@ -12,189 +12,181 @@
+        <div class="tourbus-action">
+            <button class='btc bt_info tourbus-next'>{t}Next{/t} &gt;&gt;</button>
+        </div>
+    </li>
+    <li data-el="#chart-refresh" data-focus="#chart-refresh" data-orientation="left" data-align="center" data-highlight="true">
+        <h2>{t}Refresh actions{/t}</h2>
+        <div class="tourbus-content">
+            <p>{t}You refresh the view or activate the auto-refresh.{/t}</p>
+        </div>
+        <div class="tourbus-action">
+            <button class='btc bt_info tourbus-next'>{t}Next{/t} &gt;&gt;</button>
+        </div>
+    </li>
+    <li data-el=".refresh-icon" data-focus=".refresh-icon" data-orientation="left" data-align="center" data-highlight="true">
+        <h2>{t}Refresh actions{/t}</h2>
+        <div class="tourbus-content">
+            <p>{t}Click on the icon for refresh the view.{/t}</p>
+        </div>
+        <div class="tourbus-action">
+            <button class='btc bt_info tourbus-next'>{t}Next{/t} &gt;&gt;</button>
+        </div>
+    </li>
+    <li data-el=".timer-icon" data-focus=".timer-icon" data-orientation="left" data-align="center"  data-highlight="true">
+        <h2>{t}Refresh actions{/t}</h2>
+        <div class="tourbus-content">
+            <p>{t}Click on the icon for choose a refresh interval.{/t}</p>
+        </div>
+        <div class="tourbus-action">
+            <button class='btc bt_info tourbus-next'>{t}Next{/t} &gt;&gt;</button>
+        </div>
+    </li>
+    <li data-el=".ico-col.active" data-focus=".ico-col" data-orientation="left" data-align="center" data-highlight="true">
+        <h2>{t}Grid system{/t}</h2>
+        <div class="tourbus-content">
+            <p>{t}The new performance can be display on 1, 2 or 3 columns.{/t}</p>
+            <p>{t}Use this buttons for change the number of columns.{/t}</p>
+        </div>
+        <div class="tourbus-action">
+            <button class='btc bt_info tourbus-next'>{t}Next{/t} &gt;&gt;</button>
+        </div>
+    </li>
+    <li data-orientation="centered">
+        <div class="tourbus-content">
+            <p>{t}Thanks for follow this guide.{/t}</p>
+        </div>
+        <div class="tourbus-action">
+            <button class='btc bt_info tourbus-stop'>{t}Finish{/t}</button>
+        </div>
+    </li>
 </ol>
 <div class="tourbus-overlay"></div>
+
 <script src="./include/common/javascript/moment-with-locales.js"></script>
 <script src="./include/common/javascript/moment-timezone-with-data.min.js"></script>
 <script src="./include/common/javascript/handlebars-v4.0.5.js"></script>
 <script src="./include/common/javascript/centreon/centreon-select2.js"></script>
 <script src="./include/common/javascript/jquery/tourbus/jquery-tourbus.min.js"></script>
 <script src="./include/common/javascript/jquery/jquery.cookie.js"></script>
 <script src="./include/common/javascript/centreon/centreon-tour.js"></script>
+<script src="./include/common/javascript/datepicker/localizedDatepicker.js"></script>
```

Comment:
Not enough to load it in htmlHeader.php ?

Reply:
I need to launch it again, for the popin and iFrames